include/target.mk

```diff
@@ -38,10 +38,30 @@ DEFAULT_PACKAGES+=procd-ujail
 endif
 
 # mbedTLS wireless features handling
+ifeq ($(CONFIG_TLS_PROVIDER_MBEDTLS),y)
 DEFAULT_PACKAGES+=libustream-mbedtls
 PACKAGE_NO_WIRELESS:=-wpad-basic-mbedtls
```

Comment:
The indention looks strange here, I think you should indent everything inside the ifeq block.

Reply:
Its probably highly subjective :), but I've changed that.